lib/src/sdk.ts

```diff
@@ -450,3 +455,37 @@ function getSwapStep(error: Error): string {
 
   return "UNKNOWN_STEP";
 }
+
+async function decodePayloadAndPost(
+  binaryPayload: string,
+  beData: BEData,
+  providerId: string
+) {
+  const buyApiUrl =
+    "https://buy.api.aws.prd.ldg-tech.com/sell/v1/forgeTransaction/offRamp";
```

Comment:
put this in a .env no?

Reply:
Well. I am not sure. We do not have a .env here @CremaFR, should I implement one? There are other URLs to API's as vars. Does this pose a security threat?

Reply:
I mean use .env in buy/sell and pass the url during exchangeSDK instanciation https://github.com/LedgerHQ/exchange-sdk/blob/main/lib/src/sdk.ts#L117 

we have a customUrl, I don't know if it is a swapCustomUrl tho or if it works for SELL